### DISCUSSION TITLE
[Feature] Collaborative tagging

### DISCUSSION THREAD

**Discussion by diyoyo** [immich-app/immich#10368] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In a shared album, allow the option to have a subset of "Shared People" that all members can edit to label faces collaboratively.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5089
